specification/cognitiveservices/AnomalyDetector/tspconfig.yaml

```diff
@@ -0,0 +1,26 @@
+emit: [
+  "@azure-tools/typespec-autorest",
+  "@azure-tools/typespec-python",
+  "@azure-tools/typespec-csharp",
+]
+options:
+  "@azure-tools/typespec-autorest":
+    "output-file": "openapi.json"
```

Comment:
Is this the path we want? Shouldn't we have this output into the correct path automatically?

Reply:
This PR is for typespec renaming. Ignore this now.